lib/ruby_indexer/lib/ruby_indexer/collector.rb

```diff
@@ -141,8 +141,18 @@ def handle_constant_write_node(node)
 
     sig { params(node: Prism::CallNode).void }
     def handle_call_node(node)
-      message = node.message
-      handle_private_constant(node) if message == "private_constant"
+      message = node.name
```

Comment:
We may want to check that the receiver is `null` or explicit `self` because Ruby is fun:

```rb
class Foo; end

class Bar
   Foo.attr_reader :foo
end

Bar.foo # => undefined method `foo' for Bar:Class (NoMethodError)
```